src/Worker/Delivery.php

```diff
@@ -562,5 +574,9 @@ private static function deliverMail($cmd, $contact, $owner, $target_item)
 		}
 
 		Email::send($addr, $subject, $headers, $target_item);
+
+		Model\ItemDeliveryData::incrementQueueFailed($target_item['id']);
```

Comment:
Why queue failed?

Reply:
Because of "copy and paste" without thinking :-)

Reply:
Tsk tsk tsk...